openslides/__main__.py

```diff
@@ -104,9 +104,9 @@ def get_parser():
         help='The used settings file. The file is created, if it does not exist.')
     subcommand_start.set_defaults(callback=start)
     subcommand_start.add_argument(
-        '--development',
+        '--local-installation',
         action='store_true',
-        help='Option for development purposes.')
+        help='Store settings and user files in a local directory.')
```

Comment:
Really? I tested the following:
1. `python manage.py start --local-installation` -> `openslides_data` created, server starts with this new db.
2. stop server and start again: `python manage.py start` -> same db is used (from `openslides_data`

So it seems that I need the `--local-installation` option for create only... isn't it?